.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - '8.14.0'
+  - '12.13.0'
```

Comment:
@plbabin Will this be the reason for the travis' failures? is there a reason not to upgrade it?

Reply:
yeah that's probably the reason, dunno why it was not updated 🤷